From c96b1bb728fb3e95b37990d6d44e604646a1a4f6 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me.cafebabe@gmail.com>
Date: Fri, 26 May 2023 21:06:39 +0800
Subject: [PATCH 09/16] input: dt2w: Stop using deprecated cputime64_t

---
 drivers/input/touchscreen/doubletap2wake.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/doubletap2wake.c b/drivers/input/touchscreen/doubletap2wake.c
index d30000b47..74c72409c 100644
--- a/drivers/input/touchscreen/doubletap2wake.c
+++ b/drivers/input/touchscreen/doubletap2wake.c
@@ -32,7 +32,6 @@
 #include <linux/input.h>
 #include <linux/hrtimer.h>
 #include <linux/display_state.h>
-#include <asm-generic/cputime.h>
 
 /* uncomment since no touchscreen defines android touch, do that here */
 //#define ANDROID_TOUCH_DECLARED
@@ -63,7 +62,7 @@ MODULE_LICENSE("GPLv2");
 
 /* Resources */
 int dt2w_switch = DT2W_DEFAULT;
-static cputime64_t tap_time_pre = 0;
+static u64 tap_time_pre = 0;
 static int touch_x = 0, touch_y = 0, touch_nr = 0, x_pre = 0, y_pre = 0;
 static bool touch_x_called = false, touch_y_called = false, touch_cnt = true;
 static bool exec_count = true;
-- 
2.55.0

