From fb5b8133a4aec213cf4549b03ebce1cb07a98e04 Mon Sep 17 00:00:00 2001
From: Ido Ben-Hur <idoybh2@gmail.com>
Date: Sat, 21 Dec 2024 20:19:58 +0200
Subject: [PATCH 3/6] soong: Add out/soong/*.glob* to installclean

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 ui/build/cleanbuild.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ui/build/cleanbuild.go b/ui/build/cleanbuild.go
index 065bffe..c29d1d2 100644
--- a/ui/build/cleanbuild.go
+++ b/ui/build/cleanbuild.go
@@ -106,6 +106,8 @@ func installClean(ctx Context, config Config) {
 		return filepath.Join(productOutPath, path)
 	}
 
+	removeGlobs(ctx, filepath.Join(config.OutDir(), "soong", "*.glob*"))
+
 	// Host bin, frameworks, and lib* are intentionally omitted, since
 	// otherwise we'd have to rebuild any generated files created with
 	// those tools.
-- 
2.48.1

